instances/technique/nonrigidMotionCorrection.jsonld

```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/technique/nonrigidMotionCorrection",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Technique",
+  "definition": null,
+  "description": null,
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "nonrigid motion correction",
```

Comment:
This is lacking the "-" between "non" and "rigid" (or a space, but I think that would be weird). The `@id` need to be adjusted accordingly.

Reply:
https://www.merriam-webster.com/dictionary/nonrigid should be correct